### ISSUE TITLE
[DOCS] <short description>Enhancement of UI preview section in README.md

### ISSUE THREAD

**Issue by DEEksha-ux** [Jagrati3/Environment_Animal_Safety_Hub#666]
## 📝 Description

The UI preview section of README.md needs updation.

---

## 📚 Current Issue

Images are not visible, and there is no meaningful alternate text. Instead, 'alt text' is written as a link as an alternate text for all 3 images. Clicking it yields a page not found error.

---

## 🧠 Suggested Update

If the images are not visible due to some issue in the code, it should be fixed. Also some meaningful alternate text should be provided which describes the content of the images.

---

## ✅ Checklist

- [ ] I have checked if this documentation section already exists.  -yes
- [ ] I have clearly described what needs to be updated or added.  -yes

**Comment by DEEksha-ux** [comment]
Sure
